src/godot_fmod.cpp

```diff
@@ -273,16 +273,21 @@ void Fmod::setListenerAttributes() {
             ERROR_CHECK(system->setListenerWeight(i, 0));
             continue;
         }
-        auto *ci = Object::cast_to<CanvasItem>(listener->gameObj);
+        Node *node = listener->gameObj;

```

Comment:
I'd like to use cpp 11 init now, even if the former code does not use it, so that we have less things to clean later.